bbb-common-web/src/main/java/org/bigbluebutton/presentation/DocumentConversionServiceImp.java

```diff
@@ -72,11 +72,11 @@ public void processDocument(UploadedPresentation pres) {
       } else if (SupportedFileTypes.isImageFile(fileType)) {
         imageToSwfSlidesGenerationService.generateSlides(pres);
       } else {
-
+          log.warn("Supported file not handled");

```

Comment:
Can you please add more info about the slides on this log? MeetingId, PresId, Filename, etc. if available. This way we are able to track the file and investigate more.